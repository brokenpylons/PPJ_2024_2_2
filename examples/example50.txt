((x^2 + y2) / (z^(1/2))) - (3*p - 4*q) / (2.7^2) + 5.2 * r / s^3
