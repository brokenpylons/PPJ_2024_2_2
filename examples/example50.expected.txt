lparen("(") lparen("(") variable("x") pow("^") real("2") plus("+") variable("y2") rparen(")") divides("/") lparen("(") variable("z") pow("^") lparen("(") real("1") divides("/") real("2") rparen(")") rparen(")") rparen(")") minus("-") lparen("(") real("3") times("*") variable("p") minus("-") real("4") times("*") variable("q") rparen(")") divides("/") lparen("(") real("2.7") pow("^") real("2") rparen(")") plus("+") real("5.2") times("*") variable("r") divides("/") variable("s") pow("^") real("3") 
